src/config.rs

```diff
@@ -262,6 +264,10 @@ impl RecertConfig {
             Some(value) => Some(value.as_str().context("etcd_endpoint must be a string")?.to_string()),
             None => None,
         };
+        let encryption_config = match value.remove("encryption_config") {
```

Comment:
Similarly to our cert/key parameters, we should allow the user to _also_ simply paste the contents of the encryption config directly into the parameter, instead of as a file path - this makes it easier for lifecycle agent to provide recert with this config, without having to worry about annoying temporary files to put the encryption config in (especially annoying because it launches recert inside a container)

Reply:
Good point, just today I tried adding this arg to LCA and it was indeed very very annoying :) I'll refactor this, thanks for the validation :+1:

Reply:
Updated.